### ISSUE TITLE
Attach person to local event

### ISSUE THREAD

**Issue by joewhitsitt** [uiowa/uiowa#2487]
# Epic
https://github.com/uiowa/uiowa/issues/2212

# Story

As a site editor, I want to be able to relate a counselor (person) to one or more events on my website. This allows site visitors identify with specific counselors based on the event they are viewing.

# Proposed solution

- Blocked by https://github.com/uiowa/uiowa/issues/2484 and https://github.com/uiowa/uiowa/issues/2396

**Comment by joewhitsitt** [comment]
Scott said that this relationship is no longer needed.
